### PR TITLE
update tispark-deploy

### PR DESCRIPTION
**Pull request by shiyuhang0**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

在 `TiSpark 部署拓扑` 中增加 `版本选择` 与 `配置文件` 章节

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)

### What is the related PR or file link(s)?

- This PR should be merged after https://github.com/pingcap/tiup/pull/1965
### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@shiyuhang0 是否为 v6.2 特性呢？

File: tispark-deployment-topology.md [hunk 1]
```diff
@@ -44,3 +44,73 @@ aliases: ['/docs-cn/dev/tispark-deployment-topology/']
 TiUP 不提供自动安装 JRE 的支持，该操作需要用户自行完成。JRE 8 的安装方法可以参考 [OpenJDK 的文档说明](https://openjdk.java.net/install/)。
 
 如果部署服务器上已经安装有 JRE 8，但不在系统的默认包管理工具路径中，可以通过在拓扑配置中设置 `java_home` 参数来指定要使用的 JRE 环境所在的路径。该参数对应系统环境变量 `JAVA_HOME`。
+
+> **注意：**
```
Comment (Oreoxmt) [inline tispark-deployment-topology.md:48]:
```suggestion
> **注意：**
>
```

**Comment by Oreoxmt** [comment]
> @TomShawn I don't understand the reason of the failed verify, could you please explain it

@shiyuhang0 [MANUAL LINE BREAKS: L72, L87, L104](https://ci.pingcap.net/blue/organizations/jenkins/common-check/detail/common-check/55458/pipeline/#step-48-log-77), [Lists should be surrounded by blank lines, blanks-around-fences, no-multiple-blanks](https://ci.pingcap.net/blue/organizations/jenkins/common-check/detail/common-check/55455/pipeline/#step-48-log-76)

File: tispark-deployment-topology.md [hunk 2]
```diff
@@ -44,3 +44,73 @@ aliases: ['/docs-cn/dev/tispark-deployment-topology/']
...
+
+spark-env.sh
+```
+SPARK_MASTER_HOST=${ip}
+SPARK_MASTER_PORT=7077
+SPARK_MASTER_WEBUI_PORT=8080
+SPARK_PUBLIC_DNS=${ip}
+```
+
+spark-default.conf
```
Comment (Oreoxmt) [inline tispark-deployment-topology.md:89]:
```suggestion
spark-default.conf

```

**Comment by shiyuhang0** [comment]
thank you for your review, but we decide to give up the code change as well as the doc

**Pull request closed without merging**